scripts/gis-material-update/validate-deploy/modules/common.py

```diff
@@ -0,0 +1,91 @@
+import math
+from sqlalchemy import create_engine, text
+    
+def deploy(pg_conn_uri, tormays_table_org, tormays_table_temp, logger=None):
+    # validate data amount: is it between given limits
+    delete_result = None
+    engine = create_engine(pg_conn_uri)
+    with engine.connect() as connection:
+        transaction = connection.begin()
+        delete_sql = "delete from " + tormays_table_org
+        try:
+            logger.info("Deploy ...")
+            delete_result = connection.execute(text(delete_sql))
+            logger.info(tormays_table_org + ": " + f"{delete_result.rowcount} rows deleted.")
```

Comment:
Using both concatenation and formatting on the same line is a bit funny.
```suggestion
            logger.info(f"{tormays_table_org}: {delete_result.rowcount} rows deleted.")
```

Overall, I think the formatting style is clearer than concatenation.

Reply:
Changed